org.eclipse.buildship.core.test/META-INF/MANIFEST.MF

```diff
@@ -10,11 +10,19 @@ Comment: junit has to be a required bundle, otherwise the tests won't launch for
 Require-Bundle: org.junit
 Bundle-ClassPath: .,
  lib/cglib-nodep-3.1.jar,
+ lib/commons-lang-2.6.jar,
+ lib/commons-codec-1.6.jar,
  lib/groovy-all-2.0.5.jar,
  lib/guava-18.0.jar,
  lib/hamcrest-core-1.3.jar,
+ lib/hamcrest-library-1.3.jar,
+ lib/jetty-6.1.25.jar,
+ lib/jetty-util-6.1.25.jar,
```

Comment:
I've just noticed that there are some dependencies in the _lib_ folder that are not necessary. I'll take care of those.